src/Driver.php

```diff
@@ -168,7 +168,7 @@ public function rmdir(string $path): Promise;
      * Dot entries are not included in the resulting array (i.e. "." and "..").
      *
      * @param string $path
-     * @return \Amp\Promise
+     * @return \Amp\Promise<array<int, string>>
```

Comment:
This should probably be `Promise<list<string>>`?

Reply:
Do you know if PHPStan understand list type? I'm unsure whether it's psalm's only feature or not.

Reply:
ping @ondrejmirtes

Reply:
okay, apparently it does: https://phpstan.org/r/e440dbc3-1f34-491f-9629-f0639d98fe4d

Reply:
```suggestion
     * @return \Amp\Promise<list<string>>
```